src/framework/script/script-attributes.js

```diff
@@ -148,6 +146,56 @@ function rawToValue(app, args, value, old) {
     return value;
 }
 
+/**
+ * @typedef {Object} AttributeSchema
+ * @property {"boolean"|"number"|"string"|"json"|"asset"|"entity"|"rgb"|"rgba"|"vec2"|"vec3"|"vec4"|"curve"} type - The Attribute type
+ * @property {boolean} [array] - True if this attribute is an array of `type`
+ * @property {number} [step] - The precision of the attribute. (Only necessary for numerical attributes)
+ * @property {number} [min] - The expected minimum range of the attributes (Only necessary for numerical attributes)
+ * @property {number} [max] - The expected minimum range of the attributes (Only necessary for numerical attributes)
```

Comment:
are `step`, `min` and `max` members used by the engine?

Reply:
nope. removing